versioned_docs/version-7.x/native-stack-navigator.md

```diff
@@ -193,6 +197,31 @@ Style object for header back title. Supported properties:
 
 Only supported on iOS.
 
+<img src="/assets/7.x/native-stack/headerBackTitleStyle.png" width="500" alt="Header back title style" />
+
+Example:
+
+```js
+export function NativeStack() {
+  return (
+    <Stack.Navigator>
+      <Stack.Screen
+        name="NewsFeed"
+        component={NewsFeedScreen}
+        options={{
+          title: 'Feed',
+          headerBackTitle: 'Back title example',
+          headerBackTitleStyle: {
+            fontSize: 14,
+            fontFamily: 'Georgia',
+          },
+        }}
+      />
+    </Stack.Navigator>
+  );
+}
```

Comment:
If you want to put a full example here you'd also have to add Static API (your example is in Dynamic API) see https://reactnavigation.org/docs/7.x/native-stack-navigator#usage for reference

Here. maybe a simpler example would also work just fine?

```suggestion
headerBackTitleStyle: {
  fontSize: 14,
  fontFamily: 'Georgia',
},
```